extension/src/video.js

```diff
@@ -2,16 +2,41 @@ import Binding from './services/Binding';
 
 window.addEventListener('load', (event) => {
     const bindings = [];
+    const pages = [];
+    const urlObj = new URL(window.location.href);
+
     let videoSelectMode = false;
+    let subSyncAvailable = false;
+
+    switch (urlObj.host) {
+        case 'www.netflix.com':
+            subSyncAvailable = true;
+            pages.push(chrome.runtime.getURL('pages/netflix-page.js'));
+            break;
+        case 'www.youtube.com':
+            subSyncAvailable = true;
+            pages.push(chrome.runtime.getURL('pages/youtube-page.js'));
+            break;
+        default:
+            break;
+    }
+
+    for (let index = 0, length = pages.length; index < length; index++) {
+        const s = document.createElement('script');
+
+        s.src = pages[index];
+        s.onload = () => s.remove();
+        (document.head || document.documentElement).appendChild(s);
+    }
 
     const interval = setInterval(() => {
         const videoElements = document.getElementsByTagName('video');
 
         for (const v of videoElements) {
             const bindingExists = bindings.filter((b) => b.video.isSameNode(v)).length > 0;
 
-            if (!bindingExists) {
-                const b = new Binding(v);
+            if (!bindingExists && (v.src || [...v.children].some((elm) => 'SOURCE' === elm.tagName && elm.src))) {
```

Comment:
A little bit of a micro-optimization, but I think it could be worth it since this loop runs every second.
Could you extract this test to a function? 

``` (v.src || [...v.children].some((elm) => 'SOURCE' === elm.tagName && elm.src)```

And instead of using `[...v.children].some`, just do a loop that checks each element for a source.